apps/client/src/pages/WrapperPage.js

```diff
@@ -1,141 +1,64 @@
-          questionIndex: sessionContext[slug].questionIndex - 1,
-        },
-      ]);
-    }
-    // Go to "Location"
-  };
-
-  const onGoToQuestion = (questionIndex) => {
-    // Go to the specific question in the stack
-    checker.rewindTo(questionIndex);
-    sessionContext.setSessionData([
-      slug,
-      {
-        questionIndex,
-      },
-    ]);
-  };
-
+import Location from "../components/Location/Location";
+import Questions from "../components/Questions";
+import {
+  StepByStepItem,
+  StepByStepNavigation,
+} from "../components/StepByStepNavigation";
+const WrapperPage = () => {
+  const [finishedLocation, setFinishedLocation] = useState(false);
+  const [finishedQuestions, setFinishedQuestions] = useState(false);
   return (
     <Layout>
       <Helmet>
         <title>Wrapper Page</title>
       </Helmet>
-      {checker.stack.map((q, i) => {
-        if (q === checker.stack[sessionContext[slug].questionIndex]) {
-          return (
-            <Question
-              question={q}
-              key={`question-${q.id}-${i}`}
-              onSubmit={onQuestionNext}
-              onGoToPrev={onQuestionPrev}
-              showPrev
-              showNext
-            />
-          );
-        } else {
-          let answer;
-          if (q.options) {
-            answer = q.answer;
-          } else {
-            const responseObj = booleanOptions.find(
-              (o) => o.value === q.answer
-            );
-            answer = responseObj?.label;
-          }
-          return (
-            <>
-              <Heading forwardedAs="h3">{q.text}</Heading>
-              <Paragraph>
-                {answer?.replace(/['"]+/g, "")}
-                <Button
-                  style={{ marginLeft: 20 }}
-                  onClick={() => onGoToQuestion(i)}
-                  variant="textButton"
-                >
-                  Wijzig
-                </Button>
-              </Paragraph>
-            </>
-          );
-        }
-      })}
-      <DebugDecisionTable {...{ topic, checker }} />
+      <StepByStepNavigation
+        customSize
+        disabledTextColor="inherit"
+        doneTextColor="inherit"
+        highlightActive
+        style={{ margin: "40px 0" }}
+      >
+        <StepByStepItem
+          active={!finishedLocation}
+          checked={finishedLocation}
+          heading="Adres gegevens"
+          largeCircle
+        >
+          <Location
+            finishedLocation={finishedLocation}
+            setFinishedLocation={setFinishedLocation}
+          />
+        </StepByStepItem>
+        <StepByStepItem
+          active={finishedLocation && !finishedQuestions}
+          checked={finishedQuestions}
+          heading="Vragen"
+          largeCircle
+        />
+        <Questions
+          finishedLocation={finishedLocation}
+          finishedQuestions={finishedQuestions}
+          setFinishedLocation={setFinishedLocation}
+          setFinishedQuestions={setFinishedQuestions}
```

Comment:
I would prefer to use different naming and props here, but maybe we should discuss this next dev meeting. Keep it as it is for now.